src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -238,7 +238,7 @@ void BaseQNAcceleration::updateDifferenceMatrices(
         // insert column deltaR = _residuals - _oldResiduals at pos. 0 (front) into the
         // QR decomposition and update decomposition
 
-        //apply scaling here
+        // apply scaling here
```

Comment:
```suggestion
        //apply scaling here
```
Would not add these kind of changes here. This just makes the diff unnecessary large. @fsimonis Can we add this to our linter to have one rule that is enforced consistently?